chapters/sampling-randomization-power.tex

```diff
@@ -1,391 +1,434 @@
-\index{randomization inference}
-allows us to execute a given randomization program repeatedly,
-visualize how the randomization might have gone differently,
-and calculate alternative p-values against null hypotheses.
-These \textbf{randomization inference}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Randomization\_Inference}} significance levels may be very different
-than those given by asymptotic confidence intervals,
-particularly in small samples (up to several hundred clusters).
-
-After generating the ``true'' treatment assignment,
-\texttt{ritest} illustrates the distribution of correlations
-that randomization can spuriously produce
-between \texttt{price} and \texttt{treatment}.
-
-\codeexample{randomization-program-2.do}{./code/randomization-program-2.do}
-
-\subsection{Clustering and stratification}
-
-To control randomization noise, we often use techniques
-\index{clustering}\index{stratification}
-that reduce the likelihood of a ``bad draw''.\cite{athey2017econometrics}
-These techniques can be used in any random process,
-including sampling; their implementation is nearly identical in code.
-We mean this in a specific way: we want to exclude
-randomizations with certain correlations,
-or we want to improve the \textbf{balance}
-of the average randomization draw.\cite{bruhn2009pursuit}
-The most common is \textbf{stratification},\sidenote{\url{https://dimewiki.worldbank.org/wiki/Stratified_Random_Sample}}
-which splits the sampling frame into ``similar'' groups -- \textbf{strata} --
-and randomizes \textit{within} each of these groups.
-It is important to preserve the overall likelihood for each unit to be included,
-otherwise statistical corrections become necessary.
-For a simple stratified randomization design,
-it is necessary to include strata \textbf{fixed effects},
-or an indicator variable for each strata, in any final regression.
-This accounts for the fact that randomizations were conducted within the strata,
-by comparing each unit to the others within its own strata.
-
-However, manually implementing stratified randomization
-is prone to error: in particular, it is difficult to precisely account
-for the interaction of group sizes and multiple treatment arms,
-particularly when a given strata can contain a small number of clusters,
-and when there are a large number of treatment arms.\cite{carril2017dealing}
-The user-written \texttt{randtreat} command
-properly implements stratification,
-with navigable options for handling common pitfalls.
-We demonstrate the use of this command here.
-
-\codeexample{randtreat-strata.do}{./code/randtreat-strata.do}
-
-Many studies collect data at a different level of observation than the randomization unit.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Unit_of_Observation}}
+Causal inference from randomization therefore depends on a specific counterfactual:
+that the units who received the treatment program might not have done so.
```

Comment:
_might_ sounds as if we are unsure of who received treatment and who didnt. What about:

_that the units who received the treatment program could just as well have been randomized to not have done so_.

Reply:
f7a79046a8d4623210b8804617979c5ae66de7fc